core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -123,6 +96,9 @@
      */
     @Deprecated
     public ClientFactoryBuilder() {
+        options.put(ClientFactoryOption.CHANNEL_OPTIONS,
+                    ClientFactoryOption.CHANNEL_OPTIONS.newValue(channelOptions));
```

Comment:
Could we start from an empty `options` so we can use the default `ClientFactoryOptions` instance when a user did not set any options, rather than creating a new `ClientFactoryOptions` instance?